aspnetcore/blazor/debug.md

```diff
@@ -269,3 +269,11 @@ protected override async Task OnInitializedAsync()
     ...
 }
 ```
+
+### Visual Studio timeout
+
+If Visual Studio throws an exception that the debug adapter failed to launch mentioning that the timeout was reached, you can adjust the timeout with a Registry setting. The following example doubles the default timeout of 30 seconds (30,000 milliseconds) to 1 minute (60,000 milliseconds):
```

Comment:
I believe the default timeout is now 60000 milliseconds so we might want to change the language here. Alternatively, we can avoid mentioning the default so that we don't have to update this documentation in reaction to changes in VS.